### PR TITLE
KFSPTS-12176 adding DVs to the create accounting document job

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

**Comment by jhulslander** [comment]
forgot to mention, this is ready for review

File: src/test/java/edu/cornell/kfs/fp/batch/xml/fixture/CuDisbursementVoucherDocumentNonEmployeeTravelExpenseFixture.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+public enum CuDisbursementVoucherDocumentNonEmployeeTravelExpenseFixture {
+    DELTA("A", "DELTA", 55.55),
+    OTHER_LODGING("L", "Other - Lodging", 1555.55),
+    PREPAID_AVIS("PR", "AVIS", 23.55),
+    PREPAID_OTHER("PO", "PREPAID OTHER", 230.55);
+    
+
```
Comment (ajd299) [inline src/test/java/edu/cornell/kfs/fp/batch/xml/fixture/CuDisbursementVoucherDocumentNonEmployeeTravelExpenseFixture.java]: Is the extra newline here intentional?

File: src/main/java/edu/cornell/kfs/fp/batch/service/impl/CuDisbursementVoucherDocumentGenerator.java [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+    
+    public void setUniversityDateService(UniversityDateService universityDateService) {
+        this.universityDateService = universityDateService;
+    }
+
+    public void setDisbursementVoucherTravelService(DisbursementVoucherTravelService disbursementVoucherTravelService) {
+        this.disbursementVoucherTravelService = disbursementVoucherTravelService;
+    }
+
+    public void setvendorService(VendorService vendorService) {
```
[the remark this answers is not in the document]
Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/fp/batch/service/impl/CuDisbursementVoucherDocumentGenerator.java]: odd, I am surprised Spring didn't have a problem with that.  I will fix it.

File: src/test/java/edu/cornell/kfs/fp/batch/xml/fixture/CuDisbursementVoucherDocumentNonEmployeeTravelExpenseFixture.java (hunk 1 above)
Comment (jhulslander) [reply src/test/java/edu/cornell/kfs/fp/batch/xml/fixture/CuDisbursementVoucherDocumentNonEmployeeTravelExpenseFixture.java]: no good reason, I will remove it.

**Pull request merged**